quickwit-inc/quickwit
# Remove DocMapperType using TypeTag

### Context / purpose
The DocMapperType was only used for serialization. It needs to be replaced with DocMapper.
[See Issue 85](https://github.com/quickwit-inc/quickwit/issues/85)

### Description
Added changes to use typetag crate for serializing the DocMapper Trait

### How was this PR tested?
- Run the create index command
- Verify the created index metadata jso file.

### Checklist
- [x] included documentation
